packages/frontend/src/hooks/useBigNumberConfig.ts

```diff
@@ -231,15 +232,7 @@ const useBigNumberConfig = (
     const bigNumber = !isNumber(item, firstRowValueRaw)
         ? selectedField &&
           resultsData?.rows?.[0]?.[selectedField]?.value.formatted
-        : formatValue(firstRowValueRaw, {
-              format: isField(item) ? item.format : undefined,
-              round: bigNumberStyle
-                  ? 2
-                  : isField(item)
-                  ? item.round
-                  : undefined,
-              compact: bigNumberStyle,
-          });
```

Comment:
I I suppose the big number configuration won't work if we remove these overrides.

Reply:
You were right, I guess I didn't test it enough. Fixed